actors/power/src/state.rs

```diff
@@ -79,6 +78,7 @@ pub struct State {
     /// Claimed power for each miner.
     pub claims: Cid, // Map, HAMT[address]Claim
 
+    // Deprecated as of FIP 0084
     pub proof_validation_batch: Option<Cid>,
```

Comment:
do we expect this to be empty after the upgrade? is there a period where some may get stuck in here across the upgrade epoch boundary, or are they run within the same epoch?

Reply:
Good question.  They are always pulled out in the same epoch so there shouldn't be any trouble.  I should double check cron logic in case null blocks mess with this.